staking/types/validator.go

```diff
@@ -200,10 +201,8 @@ func (v *Validator) SanityCheck(oneThirdExtrn int) error {
 		return errNilMaxTotalDelegation
 	}
 
-	// if I'm not banned, then I must
-	// ensure that MinSelfDelegation >= 1 ONE
-	if v.EPOSStatus != effective.Banned &&
-		v.MinSelfDelegation.Cmp(big.NewInt(denominations.One)) < 0 {
+	// MinSelfDelegation must be >= 1 ONE
+	if v.MinSelfDelegation.Cmp(big.NewInt(denominations.One)) < 0 {
```

Comment:
are you sure to take away ` v.EPOSStatus != effective.Banned`

Reply:
yea. basically MinSelfDelegation rule should still apply.  Slashing doesn't change it

Reply:
You don’t want to do anything with banned in sanity check ?

Reply:
Banned is used to avoid checking stake vs MinSelfStake. But it's not needed for checking minSelfStake vs 1ONE.